modules/quality_control/jsx/qualityControlIndex.js

```diff
@@ -0,0 +1,238 @@
+        type: 'select',
+        options: ImgOptions.tarchive,
+      }},
+      {label: 'Scan Location', show: true, filter: {
+        name: 'scanLocation',
+        type: 'select',
+        options: ImgOptions.scanLocation,
+      }},
+      {label: 'QC Status', show: true, filter: {
+        name: 'qCStatus',
+        type: 'select',
+        options: ImgOptions.qCStatus,
+      }},
+      {label: 'Uploaded By', show: true, filter: {
+        name: 'uploadedBy',
+        type: 'select',
+        options: ImgOptions.uploadedBy,
+      }},
+      {label: 'Selected', show: true, filter: {
+        name: 'selected',
+        type: 'select',
+        options: ImgOptions.selected,
+      }},
+      {label: 'CommentID', show: false, filter: {
+        options: ImgOptions.hidden,
+      }},
+      {label: 'TarchiveID', show: false, filter: {
+        options: ImgOptions.hidden,
+      }},
+    ];
+
+    const BehavioralFields = [
+      {label: 'Visit Label', show: true, filter: {
+        name: 'visitLabel',
+        type: 'select',
+        options: BehavioralOptions.visits,
+      }},
+      {label: 'DCCID', show: true, filter: {
+        name: 'candId',
+        type: 'text',
+      }},
+      {label: 'PSCID', show: true, filter: {
+        name: 'pSCID',
+        type: 'text',
+      }},
+      {label: 'Instrument', show: true, filter: {
+        name: 'instrument',
+        type: 'select',
+        options: BehavioralOptions.instrument,
+      }},
+    ];
+
+    let tab0 = (
+        <TabPane TabId={tabList[0].id}>
+          <FilterableDataTable
+                  name="quality_control_behavioural"
+                  data={this.state.BehavioralData.Data}
+                  fields={BehavioralFields}
+                  getFormattedCell={this.formatColumn}
+          />
+        </TabPane>
+    );
+
+    let tab1 = (
+        <TabPane TabId={tabList[1].id}>
+            <FilterableDataTable
+                name="quality_control"
+                data={this.state.ImgData.Data}
+                fields={ImgFields}
+                getFormattedCell={this.formatColumn}
+            />
+        </TabPane>
+    );
+    return (
+        <div>
+            <Tabs id = "TabPanes" tabs={tabList} defaultTab={tabList[0].id}
+                  updateURL={true}>
+                {tab0}
+                {tab1}
+            </Tabs>
+        </div>
+    );
+  }
+}
+
+QualityControlIndex.propTypes = {
+    ImgDataURL: PropTypes.string.isRequired,
+    BehavioralDataURL: PropTypes.string.isRequired,
+    hasPermission: PropTypes.func.isRequired,
+};
+
+window.addEventListener('load', () => {
+    ReactDOM.render(
+        <QualityControlIndex
+            ImgDataURL={`${loris.BaseURL}/quality_control/?format=json`}
+            BehavioralDataURL = {`${loris.BaseURL}/quality_control/quality_control_behavioral/?format=json`}
+            hasPermission={loris.userHasPermission}
+        />,
+        document.getElementById('lorisworkspace')
+    );
+});
+
```

Comment:
This file has some strange indentation. Make sure to keep jsx to 2 space indentation.

Reply:
There's still a few indentation issues. Particularly on lines `177-184`, `188-195`, `198-204`, `210-212` and `216`-223`. Other than that, this is good to go!